src/FakeItEasy/A.of.T.cs

```diff
@@ -8,7 +8,7 @@ namespace FakeItEasy
     /// Provides an API entry point for constraining arguments of fake object calls.
     /// </summary>
     /// <typeparam name="T">The type of argument to validate.</typeparam>
-    [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = nameof(A), Justification = "It is spelled correctly.")]
+    /// <remarks>If desired, <see cref="An{T}"/> may be used to specify constraints on types whose names begin with a vowel sound.</remarks>

```

Comment:
I took a page from you in MakeItEasy (but didn't use `<seealso>`, because I don't think it's supposed to contain text)

Reply:
Maybe yours is better. The docs for these docs are lacking.